cmd/jujud/agent/machine.go

```diff
@@ -483,6 +485,18 @@ func (a *MachineAgent) Run(*cmd.Context) error {
 	}
 
 	agentConfig := a.CurrentConfig()
+	machineLock, err := machinelock.New(machinelock.Config{
+		AgentName:   a.Tag().String(),
+		Clock:       clock.WallClock,
+		Logger:      loggo.GetLogger("juju.machinelock"),
+		LogFilename: agent.MachineLockLogFilename(agentConfig),
+	})
+	// There will only be an error if the required configuration
```

Comment:
NIce! This would be a great precedence to set - describe the errors to show that we understand what is happening and what we catered for, especially when we have a generic, catch-all err != nil... I really like this!!